src/components/NewDraftWizard/WizardComponents/WizardObjectIndex.js

```diff
@@ -9,10 +9,11 @@ import List from "@material-ui/core/List"
 import ListItem from "@material-ui/core/ListItem"
 import ListItemText from "@material-ui/core/ListItemText"
 import { makeStyles, withStyles } from "@material-ui/core/styles"
+import Tooltip from "@material-ui/core/Tooltip"
 import Typography from "@material-ui/core/Typography"
-import NoteAddIcon from "@material-ui/icons/NoteAdd"
+import DescriptionRoundedIcon from "@material-ui/icons/DescriptionRounded"
 import { useDispatch, useSelector } from "react-redux"
-
+import { ObjectSubmissionTypes } from "constants/wizardObject"
```

Comment:
Compare to line 19 : ObjectSubmissionTypes is already imported.

Reply:
Good call, there was some issues with merge conflicts and apparently those didn't solve that well. Will add this change on next push.